### DISCUSSION TITLE
The Service "Disable Integration" is awesome

### DISCUSSION THREAD

**Discussion by pimw1** [frenck/spook#178] [Announcements]
Hi Frenck,

The spooky integration is really handy. I see a major use case to make the service "disable an integration" (homeassistant.disable_config_entry) officaly part of home assistant: energy saving.

Let me explain my use-case, since i think my use-case is generalizable.
-When going asleep / away from home, many devices are automatically disconnected from mains power via Shelly switches to save energy. Devices like my tv, my av receiver, and my logitech harmony hub. Total power consumption is impacted a lot by doing this: a reduction of 10-15 watts.
-Until i discovered Spook, there was no way to gracefully cover this scenario in Home Assistant. You either accept warnings in the logs, or you choose not to implement energy savings (in my case, i choose to accept those warnings).

The "disable an integration"  service prevents warnings and handles this situation gracefully: some devices are really not available, and this is reflected by disabling the related integration. I've implemented the disable/enable service as part of my asleep-awake automation and come-home-go-away automation. It's valuable. And i think it should be valuable to many others.

By integrating this feature into Home Assistant, it would encourage more people to implement energy savings. Thereby making a positive impact on the environment. Especially when giving this some attention in the release notes/release party as well, this could encourage others to further embrace energy savings.

Just my thoughts, i hope you could give it some consideration. Keep up the good work!

**Comment by frenck** [comment]
I am not following this. This is Spook, not the Home Assistant project. 

Sounds like your bring this discussion to the wrong place?

../Frenck

**Comment by pimw1** [comment]
You're correct about that. I should have brought it to community.home-assistant.io > Feature Requests.

**Comment by pimw1** [comment]
I've moved the discussion to the right place: https://community.home-assistant.io/t/service-to-disable-and-enable-integrations/576101
